batch_example/batch_example.module

```diff
@@ -291,8 +291,8 @@ function batch_example_lowest_nid() {
   ->orderBy('n.nid', 'ASC')
   ->extend('PagerDefault')
   ->limit(1);
-  $nid = $select->execute()->fetchField();
-  return $nid;
+
+  return $select->execute()->fetchField();
```

Comment:
As an example it may be clearer to show that we are returning an NID, although that is documented in the code comment.